posthog/management/commands/execute_temporal_workflow.py

```diff
@@ -22,37 +22,27 @@ def add_arguments(self, parser):
         )
         parser.add_argument("--temporal_host", default=settings.TEMPORAL_HOST, help="Hostname for Temporal Scheduler")
         parser.add_argument("--temporal_port", default=settings.TEMPORAL_PORT, help="Port for Temporal Scheduler")
+        parser.add_argument("--namespace", default=settings.TEMPORAL_NAMESPACE, help="Namespace to connect to")
+        parser.add_argument("--task-queue", default=settings.TEMPORAL_TASK_QUEUE, help="Task queue to service")
         parser.add_argument(
-            "--task_queue", default=settings.TEMPORAL_TASK_QUEUE, help="Task queue to submit your workflow to"
+            "--server-root-ca-cert", default=settings.TEMPORAL_CLIENT_ROOT_CA, help="Optional root server CA cert"
         )
-        parser.add_argument("--namespace", default=settings.TEMPORAL_NAMESPACE, help="Namespace to connect to")
-        parser.add_argument("--server-root-ca-cert", help="Optional path to root server CA cert")
-        parser.add_argument("--client-cert", help="Optional path to client cert")
-        parser.add_argument("--client-key", help="Optional path to client key")
+        parser.add_argument("--client-cert", default=settings.TEMPORAL_CLIENT_CERT, help="Optional client cert")
+        parser.add_argument("--client-key", default=settings.TEMPORAL_CLIENT_KEY, help="Optional client key")
 
     def handle(self, *args, **options):
-        logging.info(f"Executing Temporal Workflow with options: {options}")
+        logging.info(f"Starting Temporal Worker with options: {options}")
```

Comment:
Why are we changing the log line? This command is ultimately executing a workflow. In fact, if no worker is running, the workflow will not run, as this doesn't start a worker.

Reply:
my bad, I just copy-pasted the whole init section, changing it